docs/catalog/web/enable-text-compression.md

```diff
@@ -0,0 +1,46 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: web
+tags: 
+ - text
+ - compression
+ - size:small
+ - role:web-developer
+---
+
+# Enable Text Compression
+
+## Description
+
+Web browsers often communicate with Web servers in a human readable format. These can be HTML, JavaScript and/or CSS - files and also REST requests which can return a response in JSON. This communication is human readable and therefore highly redudandant, therefore it can be highly compressed saving a lot of bandwidth.
+
+Web browsers and servers can communicate the compression format dynamically via the "Accept-Encoding"/"Content-Encoding" HTTP Headers. This allows the communication to dynamically switch to compression formats both side support.
+
+
+## Solution
+
+Enable all supported compression formats on the server, this allows clients to use the best available format.
+
+
+## SCI Impact
```

Comment:
I would argue this pattern also impacts the SCI's M (embodied carbon). Thoughts?

Reply:
Because the files are stored compressed? This may or may not be done, sometimes the compression is done on the fly to find the best match between client/server.
I think the change in storage SCI M embodiment would be negligible, when stored compressed. It would save much, from the website POV (maybe 10MB->5MB), but from a storage POV this is nothing as storage is measured in TB.

What SCI impact are you thinking about?

Reply:
I was thinking of transmitting compressed data similar to this [pattern](https://github.com/Green-Software-Foundation/green-software-patterns/blob/dev/docs/catalog/cloud/compress-transmitted-data.md) from an embodied carbon point of view.

Reply:
Bumping this comment again...

Reply:
Bumping this comment again? I think we should add embodied carbon point of view. Thoughts?

Reply:
Bumping comments without an actual answer to questions doesn't resolve issues.
The linked pattern has no M.

Reply:
Apologies, for the wrong link. I think what's useful as a comment for a code reiview is for the reviewer to try to help revieee to arrive at the solution, but not give the solution. And please correct me if i am incorrect, i believe that we discussed in the team meeting to include embodeid carbon in the because with the compressed sized, we are using less computing equpitment as network is reduced.